### ISSUE TITLE
[Q] font size customization

### ISSUE THREAD

**Issue by gaydenko** [mapsme/omim#831]
Hi! In standard Android (4.2.2) settings I have selected "Extra Large" to customize font size. Can map's labels follow this customization? I can't read tiny labels :(

**Comment by deathbaba** [comment]
At the moment we don’t use system settings for font size, but it is a good idea.

We have some issues with smaller texts at the moment, but they will be fixed soon.

**Comment by gaydenko** [comment]
@deathbaba , thanks for the quick answer. So, will we keep the issue open for a while?

**Comment by deathbaba** [comment]
Yes, why not? And please tell us if fonts look better in upcoming bug fixes.

**Comment by gaydenko** [comment]
OK, I will.
